### PR TITLE
Right panel search form

### PR DESCRIPTION
**Pull request by josephmturner**
also, autosize textarea wrapper

### CODE REVIEW

**Comment by josephmturner** [comment]
Fixes #146

File: src/components/RightPanelContents.tsx [hunk 1]
```diff
@@ -70,27 +65,28 @@ const RightPanelContents = (props: AllProps) => {
...
-          />
-        </StyledSvg>
+        <form onSubmit={handleSubmit}>
+          <StyledInput
+            type="text"
+            value={searchQuery}
+            onChange={handleChange}
+            darkMode={props.darkMode}
+          />{" "}
+          <StyledSvg
```
Comment (RangerMauve) [inline src/components/RightPanelContents.tsx:75]: Could you wrap this in a `<button type="submit>` and get rid of the onClick? That's more in line with accessibility guidelines.

Generally if something is going to be a button, you should implement it as a styled `<button` of some sort so you can get all the default controls that buttons get for keyboard navigation and screen readers.

It'd also be good to add an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute or a [title attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) saying this is a "Search Button".

Might be good to do an accessibility review at some point, too.

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Requested changes for some accessibility improvements in the search button. Else this looks good!

File: src/components/RightPanelContents.tsx (hunk 1 above)
Comment (josephmturner) [reply src/components/RightPanelContents.tsx]: Okay! I think the latest commit resolves those issues. I like the idea of an accessibility review!

**Pull request merged**